neurokit2/signal/signal_psd.py

```diff
@@ -194,21 +221,38 @@ def _signal_psd_welch(signal, sampling_rate=1000, nperseg=None):
 # =============================================================================
 
 
-def _signal_psd_lomb(signal, sampling_rate=1000, nperseg=None, min_frequency=0, max_frequency=np.inf):
-
-    nfft = int(nperseg * 2)
-    if max_frequency == np.inf:
-        max_frequency = 20  # sanitize highest frequency
-
-    # Specify frequency range
-    frequency = np.linspace(min_frequency, max_frequency, nfft)
-    # Compute angular frequencies
-    # angular_freqs = np.asarray(2 * np.pi / frequency)
+def _signal_psd_lomb(
+    signal, sampling_rate=1000, nperseg=None, min_frequency=0, max_frequency=np.inf, normalization=True
+):
 
-    # Specify sample times
-    t = np.arange(len(signal))
+#    nfft = int(nperseg * 2)
+#    if max_frequency == np.inf:
+#        max_frequency = 20  # sanitize highest frequency
+#
+#    # Specify frequency range
+#    frequency = np.linspace(min_frequency, max_frequency, nfft)
+#    # Compute angular frequencies
+#    # angular_freqs = np.asarray(2 * np.pi / frequency)
+#
+#    # Specify sample times
+#    t = np.arange(len(signal))
+#
+#    power = np.asarray(scipy.signal.lombscargle(t, signal, frequency, normalize=True))
+    try:
+        from astropy.timeseries import LombScargle
+        if max_frequency == np.inf:
+            max_frequency = sampling_rate / 2  # sanitize highest frequency
+        t = np.arange(len(signal)) / sampling_rate
+        frequency, power = LombScargle(t, signal, normalization='psd').autopower(minimum_frequency=min_frequency, maximum_frequency=max_frequency)
```

Comment:
```suggestion
        import astropy.timeseries
        if max_frequency == np.inf:
            max_frequency = sampling_rate / 2  # sanitize highest frequency
        t = np.arange(len(signal)) / sampling_rate
        frequency, power = astropy.timeseries.LombScargle(t, signal, normalization='psd').autopower(minimum_frequency=min_frequency, maximum_frequency=max_frequency)
```

I don't like `from x import y` 😬